sig-cluster-lifecycle/README.md

```diff
@@ -154,8 +154,8 @@ Cluster API infrastructure provider for IBM Cloud
   - Slack: [#cluster-api-ibmcloud](https://kubernetes.slack.com/messages/cluster-api-ibmcloud)
   - [Mailing List](https://groups.google.com/a/kubernetes.io/g/sig-cluster-lifecycle)
 - **Meetings:**
-  - Cluster API Provider IBM Cloud office hours: [Fridays at 09:00 IST](https://zoom.us/j/9392903494) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09%3A00&tz=IST).
-    - [Meeting notes and Agenda](https://docs.google.com/document/d/1oWnqXy1VFv0E3kovQoZfS6IlVP0L4eaQsN-2HYC_6_A).
+  - Cluster API Provider IBM Cloud office hours: [Tuesdays at 10:00 IST](https://zoom.us/j/508079177) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10%3A00&tz=IST).
```

Comment:
I assume this is still a meeting in the community account(s)?

Reply:
yes, it is shared with SIG cloud provider, courtesy @elmiko